File: src/main/java/ru/java/backend/todo/todobackend/repo/StatRepository.java
package ru.java.backend.todo.todobackend.repo;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.java.backend.todo.todobackend.entity.Stat;

@Repository
public interface StatRepository extends CrudRepository<Stat, Long> {

    Stat findByUserEmail(String email); //связь один к одному
}

File: src/main/java/ru/java/backend/todo/todobackend/repo/StatRepository.kt
package ru.java.backend.todo.todobackend.repo

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import ru.java.backend.todo.todobackend.entity.Stat

@Repository
interface StatRepository : CrudRepository<Stat, Long> {
    fun findByUserEmail(email: String): Stat //связь один к одному
}